_posts/2019-05-30-bitcoin-sov-narrative.md

```diff
@@ -0,0 +1,69 @@
+
+Mises' Regression Theorem is based on the observations made by Carl Menger in his seminal work ["On the Origins of Money"](https://mises-media.s3.amazonaws.com/On%20the%20Origins%20of%20Money_5.pdf) where he describes the process of how money arose from commodities. But, what Menger did was make a historical observation, not a theoretical explanation. It was Mises who wrongly took those observations into theory with his unfortunate Regression Theorem.
+
+In an attempt to dodge the Regression Theorem debate others have retorted with Nick Szabo's views in his remarkable essay ["Shelling out: The Origins of Money"](https://nakamotoinstitute.org/shelling-out/), using the collectible and store of value arguments. Szabo's excellent work is also a historical explanation like Menger, but more detailed and precise from an anthropological point of view.
+
+I shall not spend much time rebutting the claim that Bitcoin's source of value at the very beginning was being a collectible in the literal meaning of the term. That was not what Szabo meant and it is not credible at all to say that the first Bitcoin owners demanded it because it was a whimsy, rather than demanding it because they thought it could potentially become a medium of exchange. Bitcoin's first owners knew extremely well why Bitcoin was invented for and what was its intended purpose.
+
+A popular [narrative amongst Bitcoin supporters](https://medium.com/@vijayboyapati/the-bullish-case-for-bitcoin-6ecc8bdecc1) that apply Szabo's views claim that money **always** evolves through the following stages:
+
+1. Collectible
+2. Store of value
+3. Medium of exchange
+4. Unit of account
+
+In my view, the above narrative is scientifically inconsistent and as such, it could be easily utilized against Bitcoin by academia and governments to spread negative propaganda like  accusing Bitcoin of being a reckless speculative bubble and that there is no scientific support of its value other than "It's valuable because people value it". It's not that I fear too much that they can stop anything - I believe Bitcoin will thrive or not regardless of what haters and supporters claim - though, they might be able to slow down the adoption process a bit.
```

Comment:
Remove the extra space between "like" and "accusing. Put parentheses around "or not"